src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -878,25 +859,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     ============================================================
     -->
   <Target Name="_ComputeNETCoreBuildOutputFiles"
-          DependsOnTargets="_ChooseAppHost;_GetComHostPaths;_GetIjwHostPaths"
+          DependsOnTargets="_GetAppHostPaths;_GetComHostPaths;_GetIjwHostPaths"
           BeforeTargets="AssignTargetPaths"
           Condition="'$(ComputeNETCoreBuildOutputFiles)' == 'true'">
 
-    <!-- Fallback to renaming the dotnet host if there is no apphost for self-contained builds. -->
```

Comment:
Anyone have context on this? I saw no reason for it to exist:
- We should have already errored out if we couldn't find an appropriate apphost pack
- We explicitly block running `dotnet` if it has been renamed

Reply:
For < 2.0, it seems.